services/core-api/app/api/EMLI_contacts/resources/EMLI_contact.py

```diff
@@ -0,0 +1,74 @@
+from flask import current_app
+from flask_restplus import Resource, reqparse
+from werkzeug.exceptions import NotFound, BadRequest
+from app.extensions import api
+from flask import request
+from app.api.utils.resources_mixins import UserMixin
+from app.api.utils.access_decorators import requires_role_edit_emli_contacts, requires_role_mine_admin, requires_any_of, VIEW_ALL, MINESPACE_PROPONENT
+from app.api.EMLI_contacts.response_models import EMLI_CONTACT_MODEL
+from app.api.EMLI_contacts.models.EMLI_contact import EMLIContact
+from app.api.EMLI_contacts.models.EMLI_contact_type import EMLIContactType
+
+
+class EMLIContactResource(Resource, UserMixin):
+    parser = reqparse.RequestParser()
+    parser.add_argument('first_name', type=str, trim=True, help='EMLI First name', location='json')
+    parser.add_argument('last_name', type=str, help='EMLI Last name.', location='json')
```

Comment:
Is there a reason this field is not trimmed like the first_name?

Reply:
Added trim=True to last_name